### PR TITLE
Docs(next): changed init docs , added auto flag for the node 

### PR DESCRIPTION
**Pull request by anikethsaha**
**What kind of change does this PR introduce?**

refactoring

**Did you add tests for your changes?**
NA

**If relevant, did you update the documentation?**
NA

**Summary**
as in the init package, its using `args.slice(3)` in order to detect whether any custom scaffold or `--auto` flag is passed or not, in the init docs it was 
`init([null,null,..myPackages])`

**there are two conflicts with this**

- in the `init/index.ts`, in `initializeInquirer` the arguments are spread so its actually converting it into array , so if there is `init([ ])` used to call the API, then its actually spreading it and making array of array `[ [ ] ]`, and it will create an error, as there won't be any **3rd** index so it will be skipped

- there are 3 arguments mentioned in the docs which are passing to the method, so as its `slice(3)` again the scaffold will be skipped

Also 
I added docs for running the `init` using node for `--auto` flag too

**Does this PR introduce a breaking change?**
No

**Other information**
NA

### CODE REVIEW

**Comment by anikethsaha** [comment]
cc @evenstensberg

File: packages/init/README.md [hunk 1]
```diff
@@ -25,7 +25,10 @@ const init = require("@webpack-cli/init").default;
 init();
 
 // we're slicing node.process, ...myPacakges is a webpack-scaffold name/path
-init([null, null, ...myPacakges]);
```
Comment (evenstensberg) [inline packages/init/README.md]: We actually need to support both, so this change needs to be adjusted accordingly

Comment (anikethsaha) [reply packages/init/README.md]: support both ?
cli and node ?

Comment (evenstensberg) [reply packages/init/README.md]: No we're supporting both auto flag and multiple pkgs

Comment (anikethsaha) [reply packages/init/README.md]:
```js
options = [...myPackages,"--auto"]
init(null, null, null, options);
```
like this can we add in the docs ?

Comment (evenstensberg) [reply packages/init/README.md]: Let's keep the array convention, and stick with either having a spread plus the auto flag or just the packages.

Comment (anikethsaha) [reply packages/init/README.md]: > stick with either having a spread plus the auto flag or just the packages.

thinking of keeping with the auto flag. 
 
> Let's keep the array convention, 

but then in the init method , the method argument will be an array of array. of slicing of 3 may not work

Comment (evenstensberg) [reply packages/init/README.md]: Then you can tweak the api

**Pull request closed without merging**